roles/foreman_installer/tasks/installer_pr.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: "install git"
+  package:
+    name: 'git'
+    state: 'present'
+  tags:
+    - packages
+
+- name: "fetch git PR"
```

Comment:
we have something very similar in the `module_pr` playbook:

https://github.com/theforeman/forklift/blob/bb09a2c9bf9ba0862706f12b0005504717129cd3/roles/foreman_installer/tasks/module_pr.yml#L32-L54

I wonder if we could slim down the `shell` part a bit here?